tests/acceptance/project-conversations-test.js

```diff
@@ -214,11 +212,28 @@ test('Project admin can reopen a conversation', function(assert) {
     project: project.slug
   });
 
+  andThen(() => {
+    page.statusSelect.openButton.click();
```

Comment:
The new async await would be rid of the andThens'.  I'm sure you already know that but I found it was useful to start moving acceptance tests over 1 test file at a time (maybe a codemod) available now???